src/main/java/com/google/netpcapanalysis/mockdata/MockDataLoader.java

```diff
@@ -1,33 +1,31 @@
-package com.google.sps.mockdata;
+package com.google.netpcapanalysis.mockdata;
 
-import java.io.FileNotFoundException;
 import java.io.FileReader;
 import java.io.IOException;
 import java.io.BufferedReader;
 import java.util.ArrayList;
-import java.util.Scanner;
 
-import com.google.sps.datastore.PCAPdata;
-import com.google.sps.datastore.GenericPCAPDaoImpl;
-import com.google.sps.datastore.GenericPCAPDao;
+import com.google.netpcapanalysis.models.PCAPdata;
+import com.google.netpcapanalysis.dao.PCAPDaoImpl;
+import com.google.netpcapanalysis.interfaces.dao.PCAPDao;
 
 public class MockDataLoader {
+
   private static final String FILE_NAME = "file_1";
   // CSV format:
   // Source,Destination,Domain,Location,Protocal,Size,Flagged,Frequency
   private String csvFile = "data.csv"; // CSV located in project dir /webapp
 
-  public MockDataLoader(){
+  public MockDataLoader() {
 
   }
 
   public void LoadData(ArrayList<PCAPdata> dataTable) { // uploads to datastore
-    GenericPCAPDao dataBase = new GenericPCAPDaoImpl();
+    PCAPDao dataBase = new PCAPDaoImpl();
```

Comment:
You are creating the DAO every time instead of using a factory that maintains a global instance. So could lead to problems if two people trying to call this function on your website. The PCAPDaoImpl should be modified to a factory pattern like getInstance() instead allowing a constructor. May be do it on your next iteration.

Reply:
SG